### ISSUE TITLE
[Bug]: NPC Spawn text box types backwards

### ISSUE THREAD

**Issue by TexasToast03** [YimMenu/HorseMenu#188]
### Describe the bug

When I open the NPC spawn list, it types backwards. (ex. crocodile = elidocorc)

### Steps To Reproduce

1. Go to spawn menu
2. Type in a NPC
3. Starts typing backwards. (The line that shows you where your typing moves behind the letter you typed into the spawn text box) ex. crocodile = elidocorc

### Expected Behavior

1. Go to spawn menu
2. Type in a NPC
3. Letters appear after the last letter and not before.

### Logs and/or screenshots

```
<paste your logs here>
```

### Store

Epic Games

### Language

English

### Additional context

It was working before but recently started to act up. I didn't mess with any of the files.

### Build

- [X] I've confirmed the issue exists on the latest version of HorseMenu

**Comment by Rxann** [comment]
Yeah I have noticed this. I think it might be the custom InputTextWithHint(speculation). I will look into it.

**Comment by TexasToast03** [comment]
> Yeah I have noticed this. I think it might be the custom InputTextWithHint(speculation). I will look into it.

I think it happens when I open the rstar social menu and message someone

**Comment by Rxann** [comment]
> I think it happens when I open the rstar social menu and message someone

Okay, let me test this out. You might be on to something.

**Comment by Rxann** [comment]
Confirmed this is one of the triggers to the glitch. I will start working on a fix.

**Comment by Rxann** [comment]
This might be a potential fix:
https://github.com/ocornut/imgui/issues/429#issuecomment-163145047
